### ISSUE TITLE
Allow PopUps for specific / list of websites ?

### ISSUE THREAD

**Issue by bboysathish** [brave/browser-laptop#2677]
**Did you search for similar issues before submitting this one?** Yes

**Describe the issue you encountered:** 
I am trying to access a website, which use popups to maintain session. But brave is not allowing popups, is there way to allow popups for particular website ?

**Expected behavior:** Settings to allow popups for specific website or list of websites
- Platform (Win7, 8, 10? macOS? Linux distro?): windows 8
- Brave Version: 0.11.1
- Steps to reproduce:
  Well, this is an intranet website, please use any website which uses popups to maintain session. 
- Screenshot if needed: 
  ![session](https://cloud.githubusercontent.com/assets/8023447/17076843/d9e96a0a-50da-11e6-924b-553e72a858a8.PNG)
- Any related issues: No, as I know of

**Comment by debiangirl** [comment]
This dont help? [https://github.com/brave/browser-laptop/issues/1621](https://github.com/brave/browser-laptop/issues/1621)

**Comment by luixxiul** [comment]
Are scripts allowed there?

**Comment by bbondy** [comment]
This is a known thing we don't provide an option for yet.  @bridiver might be able to dump some specifics on how to fix for someone else to pickup.

**Comment by bbondy** [comment]
It should be added into the Bravery panel though once implemented.

**Comment by luixxiul** [comment]
So is this related with #94?

**Comment by bridiver** [comment]
I don't have a specific timeline for it right now apart from 1.0

**Comment by petrocc** [comment]
I use the Zimbra for my non-professional email server, and my preferred method of reading my email is to pop it out into a separate window. Currently Brave inhibits that behavior (the first double click Zimbra warns me that popups are blocked, the second time it opens the windows). 

I looked at #1621 as linked above, and that would probably be the fix except there's no "block popups" option on the panel that I can see. 

Currently I'm using 0.11.1, electron 1.2.7 if that matters. 

Otherwise nothing things are working well. Thanks folks.

**Comment by alexwykoff** [comment]
+1 support https://linkbubble.zendesk.com/agent/tickets/5257

**Comment by bkarlan** [comment]
Has this been resolved yet?  There are certain sites I must use that require pop ups to be allowed.

**Comment by srirambv** [comment]
+1 from [support ](https://linkbubble.zendesk.com/agent/tickets/6811)

**Comment by srirambv** [comment]
+1 from [support ](https://linkbubble.zendesk.com/agent/tickets/7099) on Pearson publishing
> I’ve been using Brave with success, until I got ready to send out a class-wide announcement from MyLabsPlus; the announcement will not send; I get an error message saying that the browser is blocking pop-ups.

**Comment by aflores** [comment]
+1  I am not able to launch an online class hosted in Cornerstone OnDemand (https://www.cornerstoneondemand.com/) so back to Chrome for me :-(

**Comment by lukemulks** [comment]
+1 from Brave Community contributor

Community URL: https://community.brave.com/t/brave-needs-to-be-able-to-distinguish-the-legitimate-use-of-pop-ups/2045

>_The Web interface to certain business backend systems (an SAP system in my case) use pop-up windows as part of the normal user interaction. E.G. confirmation of user actions such as "Save, Don't Save, Cancel" or "Are you sure? Yes/No". These are blocked by Brave and mean that you cannot proceed any further with the business transaction that has otherwise been working very nicely._

**Comment by vanjan** [comment]
I've just blocked popups on one website but I need them back -- how do I re-enable them?

**Comment by NejcZdovc** [comment]
@vanjan you can do that under Preferences (`about:preferences#security`), at the bottom of that page

**Comment by bkarlan** [comment]
I've long since given up on your browser.  If it takes this long to resolve a most simple and basic request, which is to allow or not allow pop ups, then how long will it take to resolve more complex issues?  Please remove me from these threads.

**Comment by vanjan** [comment]
@NejcZdovc thank for a quick response. Unfortunately I don't see a way to do that...
![screen shot 2017-08-04 at 23 17 05](https://user-images.githubusercontent.com/130294/28989080-16d7a094-796b-11e7-8cfb-5e943b30cfd2.png)

**Comment by luixxiul** [comment]
> Please remove me from these threads.

Please click the unsubscribe button on the right column.

**Comment by bsclifton** [comment]
Closing as a duplicate of https://github.com/brave/browser-laptop/issues/3752
